packages/store/src/host-environment/host-environment.ts

```diff
@@ -1,6 +1,5 @@
 import { Inject, Injectable } from '@angular/core';
-import { NG_DEV_MODE, NG_TEST_MODE } from '../symbols';
-import { Callback } from '../internal/internals';
+import { Callback, NG_DEV_MODE, NG_TEST_MODE } from '../symbols';
```

Comment:
What is the reason for moving this from `internals.ts` to `symbols.ts`?

Reply:
done